scripts/runner.node.mjs

```diff
@@ -1891,6 +1891,27 @@ function getRelevantTests(cwd, testModifiers, testExpectations) {
     filteredTests.push(...availableTests);
   }
 
+  // Prioritize modified test files
+  if (allFiles.length > 0) {
+    const modifiedTests = new Set(
+      allFiles
+        .filter(filename => filename.startsWith("test/") && isTest(filename))
+        .map(filename => filename.slice("test/".length)),
+    );
+
+    if (modifiedTests.size > 0) {
+      return filteredTests
+        .map(testPath => testPath.replaceAll("\\", "/"))
+        .sort((a, b) => {
+          const aModified = modifiedTests.has(a);
+          const bModified = modifiedTests.has(b);
```

Comment:
these will both be false because the paths in `modifiedTests` will always have the `test/` prefix while all the paths in `filteredTests` will not